mono/utils/mono-dl-wasm.c

```diff
@@ -12,6 +12,63 @@
 #include <string.h>
 #include <glib.h>
 
+typedef struct MonoDlWasmFunctionEntry
+{
+	char *name;
+	void *ptr;
+	struct MonoDlWasmFunctionEntry *next;
+
+} MonoDlWasmFunctionEntry;
+
+typedef struct MonoDlWasmLibraryEntry
+{
+	char *name;
+	MonoDlWasmFunctionEntry *functions;
+	struct MonoDlWasmLibraryEntry *next;
+
+} MonoDlWasmLibraryEntry;
+
+
+static MonoDlWasmLibraryEntry* mono_dl_wasm_entries = NULL;
```

Comment:
Wouldn't it be better to rename 'entries' to 'libraries', so we can consistently distinguish between `MonoDlWasmLibraryEntry` and `MonoDlWasmFunctionEntry`?
Also in the local variables inside the functions.